fastlane/lib/assets/completions/completion.bash

```diff
@@ -0,0 +1,16 @@
+_fastlane_complete() {
+  COMPREPLY=()
+  local word="${COMP_WORDS[COMP_CWORD]}"
+  local completions=""
+
+  # look for Fastfile either in this directory or fastlane/ then grab the lane names
+  if [[ -e "Fastfile" ]]; then
+    completions=`cat Fastfile | grep "lane \:" | awk -F ':' '{print $2}' | awk -F ' ' '{print $1}'`
+  elif [[ -e "fastlane/Fastfile" ]]; then
+    completions=`cat fastlane/Fastfile | grep "lane \:" | awk -F ':' '{print $2}' | awk -F ' ' '{print $1}'`
```

Comment:
Pull the if/else in to chosing the directory so the chained pipes don't have to be duplicated. e.g.

```
if [[  -e "Fastfile]]; then
  directory = "."
else
  directory = ""
end

completions = "cat ..."
```